### PR TITLE
junos: Docs update

### PR DESCRIPTION
**Pull request by gundalow**
##### ISSUE TYPE
- Docs Pull Request
##### COMPONENT NAME

junos_
##### ANSIBLE VERSION

devel
##### SUMMARY

More docs updates

`O(...)` isn't used anywhere else

---

This change is [Reviewable](https://reviewable.io/reviews/ansible/ansible-modules-core/4326)

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks again to @gundalow. This module is going into community review.

We notice that you are one of the maintainers (@privateip, @gundalow) of this module, so when you think it's ready, please comment "shipit" and we will consider it for merging.

[This message brought to you by your friendly Ansibull-bot.]

File: network/junos/junos_config.py [hunk 1]
```diff
@@ -77,9 +80,12 @@
     description:
       - The C(replace) argument will instruct the remote device to
         replace the current configuration hierarchy with the one specified
-        in the corresponding hierarchy of the source configuraiton loaded
+        in the corresponding hierarchy of the source configuration loaded
         from this module.
     required: true
+    choices:
```
Comment (privateip) [inline network/junos/junos_config.py:83]: have we standardized on true, false vs yes, no for bool choices?

Comment (bcoca) [inline network/junos/junos_config.py:83]: We normally don't add 'choices' to boolean arguments, just state that it is a boolean.

YAML itself uses a list ([y, yes, on, 1, true] and case variants)  to convert into True/False, we try to match that in jinja2 and ansible module processing.

**Comment by gundalow** [comment]
> We normally don't add 'choices' to boolean arguments, just state that it is a boolean.
> 
> YAML itself uses a list ([y, yes, on, 1, true] and case variants) to convert into True/False, we try to match that in jinja2 and ansible module processing.

ergh, this is inconsistent all all over the modules in both repos, including some things listed as saying they support `B, O, O, L, E, A, N, A` _headdesk_, though I think I've fixed all of those.

So I'll remove `choices:` that I edited.

I'd be tempted to raise a PR to make this consistent across all modules, but that's a discussion we need to have in the Core Team.

**Pull request merged**